src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -11,44 +11,42 @@ namespace Microsoft.CodeAnalysis.CSharp
 {
     internal partial class SymbolDisplayVisitor
     {
-        private void VisitTypeSymbolWithAnnotations(TypeSymbolWithAnnotations type, AbstractSymbolDisplayVisitor visitor = null)
+        private void VisitTypeSymbolWithAnnotations(TypeSymbolWithAnnotations type, AbstractSymbolDisplayVisitor visitorOpt = null)
         {
-            if (visitor == null)
+            var visitor = (SymbolDisplayVisitor)(visitorOpt ?? this.NotFirstVisitor);
```

Comment:
>var visitor = (SymbolDisplayVisitor)(visitorOpt ?? this.NotFirstVisitor); [](start = 12, length = 73)

This is an absolutely unnecessary refactoring, and it doesn't make things better in my opinion. Please revert code to its original state.